cfgov/paying_for_college/disclosures/scripts/process_cohorts.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import sys
+
+import localflavor
+import numpy as np
```

Comment:
I'd like to do a more thorough review of this PR, but one quick thing that jumps out: Numpy is a pretty heavyweight package. Is there any lighter method of doing this processing that doesn't require it?

Reply:
I mistakenly thought we had numpy in our requirements. Now that you've set me straight, I'll pursue a lighter option. I'm only using it for one [percentile ranking calculation](https://github.com/cfpb/cfgov-refresh/blob/e38b9867db577ce7575e7e157e18a7aa68a20ab4/cfgov/paying_for_college/disclosures/scripts/process_cohorts.py#L54-L58).

Reply:
numpy requirement removed. I tested the replacement method against 400 random schools, and the ranking results were the same as the numpy method.